src/pages-and-resources/teams/messages.js

```diff
@@ -93,6 +93,14 @@ const messages = defineMessages({
     id: 'authoring.pagesAndResources.teams.group.types.open',
     defaultMessage: 'Open',
   },
+  groupTypeOpenManaged: {
+    id: 'authoring.pagesAndResources.teams.group.types.open_managed',
+    defaultMessage: 'Open managed',
+  },
+  groupTypeOpenManagedDescription: {
+    id: 'authoring.pagesAndResources.teams.group.types.open_managed.description',
+    defaultMessage: 'Only course staff can control teams. Learners can see other teams and choose it.',
```

Comment:
This wording came from the initial proposal and now that I'm testing it in the browser I think the message could be clearer by using the same language as the previous groups.

How about:
`"Only course staff can create teams. Learners can see, join and leave teams."` ?

Reply:
Done both changes